idaes/models_extra/power_generation/costing/power_plant_capcost_custom_dict.py

```diff
@@ -0,0 +1,676 @@
+#################################################################################
```

Comment:
I think the naming convention for this file should be more specific since we might need more custom dictionaries in the future, unless the plan is to add all new accounts to this file.

Reply:
I renamed the file `cansolv_mea_capcost_custom_dict.py` to better reflect the accounts present.